nautobot/docs/user-guide/index.md

```diff
@@ -1 +1,72 @@
+
+## Quick Links
+
+!!! Tip
+    Are you a **new user** to Nautobot looking to get started? Explore the following resources.
+<!-- pyml disable-num-lines 25 no-inline-html -->
+<div class="grid cards" markdown>
+
+- :material-cloud-download:{ .lg .middle } **Installing Nautobot**
+
+    ---
+    Get started with [Nautobot installation](./administration/installation/index.md) on your own Linux Platform or in a Docker environment.
+
+- :material-play-network:{ .lg .middle } **Using Nautobot**
+
+    ---
+    Dive into [how to use Nautobot](./feature-guides/getting-started/index.md) and the key components of the core web interface.
+
+- :material-transition-masked:{ .lg .middle } **Migrating to Nautobot**
+
+    ---
+    Dig into how you can [migrate from NetBox](./administration/migration/migrating-from-netbox.md) to Nautobot seamlessly.
+
+- :material-book-open-variant:{ .lg .middle } **New User Resources**
+
+    ---
+    Take advantage of the [Nautobot Navigators](https://go.networktocode.com/nautobot-navigators-collection) collection of curated resources, including eBooks and a guided Nautobot Sandbox tour.
+
+</div>
+
+!!! Tip
+    Already familiar with Nautobot and looking to expand your knowledge? Check out these advanced topics.
+<!-- pyml disable-num-lines 23 no-inline-html -->
+<div class="grid cards" markdown>
+
+- :material-api:{ .lg .middle } **Nautobot APIs**
+
+    ---
+    Check out the [REST API documentation](./platform-functionality/rest-api/overview.md) covering both basic features and advanced use cases.
+
+    Are you looking for [GraphQL](./platform-functionality/graphql.md) APIs instead? If you are new to GraphQL, you can also read the [Nautobot GraphQL User Guide](./platform-functionality/graphql.md).
+
+- :material-briefcase-edit-outline:{ .lg .middle } **Jobs**
+
+    ---
+    Looking to run your custom Python code on demand?
+
+    Explore [Nautobot Jobs](./platform-functionality/jobs/index.md), which leverage your Nautobot data to automate using your Python code.
+
+- :material-fire:{ .lg .middle } **Nautobot & NAPALM**
+
+    ---
+    Are you curious how [NAPALM integrates with Nautobot?](./platform-functionality/napalm.md) Learn how you can use NAPALM functionality to fetch live data from network devices.
+
+- :material-view-dashboard:{ .lg .middle } **Nautobot Apps & Solutions**
+
+    ---
+    Discover the power of [Nautobot Apps & Solutions](https://networktocode.com/nautobot/nautobot-apps/), from the Golden Configuration app enforcing configuration compliance to the SSoT app synchronizing data across multiple sources of truth.
+
+    These apps are both open source and commercially supported by Network to Code.
```

Comment:
Are we talking about all apps from https://networktocode.com/nautobot/nautobot-apps/ or just Golden Config and SSoT. If it is the former, then we probably should do:
```suggestion
    These apps are all open source and commercially supported by Network to Code.
```
It is a little ambiguous IMHO.

Reply:
Makes sense, yeah I was trying to give two examples as illustrative of all the apps, so that change is helpful.